Add Overworld test plans (#112)
Gamify-IT/docs: dev-manuals/test-plans/overworld-frontend/plans/minigame-spot-not-configurated.md

# Name: Minigame spot not configurated

Number: U.minigame.3  
Version: V0.1, 2022-08-08, create test case
Author: Florian Wüst  
Tester: -  

## Description

> This use case verifies that loading the minigame data for a not configurated minigame from the backend as well as setting up the minigame object  
> as inactive works as intended.
> 
> ## Precondition
> 
> The minigame object is configurated in the overworld (world and index are set) and the needed data exists in the backend (minigame is not configurated).  
> The game is started and the backend runs.
> 
> ## Postcondition
> 
> The minigame object is set up according to the data in the data base.
> 
> ## Typical procedure
> 
> 1. The tester walks to the minigame spot.  
> 2. The minigame spot is not displayed.
> 
> ## Alternative procedures
> 
> N/A

Reviewer: ```suggestion
This use case verifies that disabling minigames works as intended.

## Precondition

The minigame (spot) is configured as disabled.  
The game is started and the backend runs.

## Postcondition

-

## Typical procedure

1. Walk to the minigame spot
2. The minigame spot is not displayed

## Alternative procedures

2.1. The spot is displayed, but disabled
```
